File: crypto-trader-macos/Sources/Dto/CryptoDto.swift
struct CryptoDto: Decodable {
  let market: String
  let nameKr: String
  let nameEn: String
  
  func toCrypto() -> Crypto {
    Crypto(market: market, nameKr: nameKr, nameEn: nameEn, ticker: Ticker())
  }
}

File: crypto-module/api-server/src/main/java/com/crypto_trader/api_server/dto/CryptoDto.java
package com.crypto_trader.api_server.dto;

public class CryptoDto {
    private String market;
    private String nameKr;
    private String nameEn;

    public CryptoDto(String market, String nameKr, String nameEn) {
        this.market = market;
        this.nameKr = nameKr;
        this.nameEn = nameEn;
    }

    public CryptoDto(String market) {
        this.market = market;
        this.nameKr = market;
        this.nameEn = market;
    }

    public String getMarket() {
        return market;
    }

    public String getNameKr() {
        return nameKr;
    }

    public String getNameEn() {
        return nameEn;
    }
}
